### PR TITLE
Update grids-core.css

### PR DESCRIPTION
**Pull request by xlusive**
Flexbox support for modern browsers. Without prefixes.

### CODE REVIEW

**Comment by jamesalley** [comment]
I'm on it. I'm going to pull down this branch to double-check everything. If all looks good this will be in 0.6.

**Comment by jamesalley** [comment]
I'm going to leave grids to a subsequent release. I don't want to touch it right now, as this is not a critical fix and we want to get 0.6 out.

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by stilliard** [comment]
Hi, any updates on this?

File: src/grids/css/grids-core.css [hunk 1]
```diff
@@ -37,6 +37,10 @@
     /* IE10 uses display: flexbox */
     display: -ms-flexbox;
     -ms-flex-flow: row wrap;
+    
+     /* Modern browsers flexbox */
```
Comment (lkraav) [inline src/grids/css/grids-core.css]: There seems to be a white-space error here, the comment starts +1 column right

**Comment by ifraixedes** [comment]
I had to add myself this to pure 0.6.0, is there any plan to add as a patch?

**Comment by wouterhendriks** [comment]
Would be nice if this would be added to Pure

**Comment by redonkulus** [comment]
Replaced by https://github.com/yahoo/pure/pull/617

**Pull request closed without merging**